tests/unit/instance_test.py

```diff
@@ -207,6 +192,7 @@ def test_all_fact_ids():
         instant_facts=instant_facts,
         duration_facts=duration_facts,
         filing_name="test_instance",
+        taxonomy_version="form-1_2022-01-01",
```

Comment:
nit: it might be good to differentiate `taxonomy_version` in `Instance`, which doesn't expect `.zip`, from `taxonomy_version` in `InstanceBuilder`, which does.

Reply:
There all actually using the `.zip` version, fixed!